src/main/java/in/twizmwaz/cardinal/command/PunishmentCommands.java

```diff
@@ -57,8 +57,8 @@ public static void warn(CommandContext cmd, CommandSender sender) throws Command
         }
     }
 
-    @Command(aliases = {"ban", "pb"}, usage = "<player> [reason]", desc = "Ban a player.", min = 1)
-    @CommandPermissions("cardinal.punish.ban")
+    @Command(aliases = {"ban", "permaban", "pb"}, usage = "<player> [reason]", desc = "Ban a player.", min = 1)
+    @CommandPermissions("cardinal.punish.permaban")
```

Comment:
I don't see a reason to change the permission unless you are adding a /tempban <player> <time> [reason]